src/components/Icon.tsx

```diff
@@ -21,6 +21,127 @@ function CryptoIcon() {
 }
 Icon.Crypto = CryptoIcon;
 
+function Edit2({
+  width = 20,
+  height = 20,
+  className = '',
+  stroke = 'stroke-stone-950',
+}: IconProps) {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 20 20"
+      className={clsx('fill-none', className, stroke)}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M4.41091 13.0162L13.0159 4.41123C13.3409 4.08623 13.8684 4.08623 14.1934 4.41123L15.5901 5.8079C15.9151 6.1329 15.9151 6.6604 15.5901 6.9854L6.98425 15.5896C6.82841 15.7462 6.61675 15.8337 6.39591 15.8337H4.16675V13.6046C4.16675 13.3837 4.25425 13.1721 4.41091 13.0162Z"
+        stroke={stroke}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+      <path
+        d="M11.4583 5.96655L14.0333 8.54155"
+        stroke="#F6F8FA"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+}
+Icon.Edit2 = Edit2;
+
+function Play({ width = 16, height = 18, className = '', stroke = 'stroke-stone-950' }: IconProps) {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 14 16"
+      className={clsx('fill-none', className, stroke)}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M1.75 1.25L12.25 8L1.75 14.75V1.25Z"
+        stroke={stroke}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+}
+Icon.Play = Play;
+
+function Cross({
+  width = 18,
+  height = 18,
+  className = '',
+  stroke = 'stroke-stone-950',
+}: IconProps) {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 18 18"
+      className={clsx('fill-none', className, stroke)}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M13.5 4.5L4.5 13.5"
+        stroke={stroke}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+      <path
+        d="M4.5 4.5L13.5 13.5"
+        stroke={stroke}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+}
+Icon.Cross = Cross;
+
+function Pause2({
```

Comment:
Whare is the other pause? Is it still in use after this change or can we move to single pause Icon?

Reply:
Other pause icon is the icon with circular border being used in the drops list more options:
![Screenshot 2023-06-13 at 7 57 59 PM](https://github.com/holaplex/hub/assets/1254443/c6215efe-3f14-45b4-b419-fa020a0ae5ba)